### DISCUSSION TITLE
Over-the-air updates

### DISCUSSION THREAD

**Discussion by richatcil** [quasarframework/quasar#10134] [CLI - Capacitor / Cordova modes]
Is there a defined process for providing over-the-air updates (bypassing the need to publish an update to an appstore)?  I see Ionic has something called AppFlow that supports Capacitor.  I was just wondering as my client has indicated an interest.

**Comment by ontwikkelfabriek** [comment]
@richatcil 

> Is there a defined process for providing over-the-air updates (bypassing the need to publish an update to an appstore)?

There's `MS Codepush` for `react native` and `cordova` apps:
https://docs.microsoft.com/en-us/appcenter/distribution/codepush/

Personally I use `PWA's` (with auto update) to bypass the app store hassle completely.
